content/how-to/bpmn/orchestration-overview.md

```diff
@@ -0,0 +1,80 @@
+
+## Message exchange
+
+A BPMN workflow can publish and subscribe to both the Rhize message broker and external data sources.
+This provides substantial flexibility to create start conditions and to pass information between services.
+
+```mermaid
+flowchart TD
+trigger(I want to trigger a workflow)-->where_publish
+where_publish{Where to publish?} -->|Rhize broker| to_rhize(Use BPMN start event)
+where_publish -->|my broker| data_source(Model data source)
+data_source --> rule(Create rule)
+rule --> associate(Associate BPMN workflow)
+
+1(I want to subscribe to the result of a workflow) --> where_subscribe{Where to subscribe?}
+where_subscribe -->|to my broker|model_data_source(Model data source)
+model_data_source --> write_task(Use data-source-write task)
+where_subscribe --> |Rhize broker| throw(Use message throw event)
+throw --> subscribe(Subscribe to relevant broker/topic)
+write_task --> subscribe
+```
+
+For a more detailed discussion about what trigger to use, read the next section.
+
+## Triggers
+
+You also have multiple ways to {{< abbr "trigger" >}} a BPMN.
+The best choice of trigger depends on the context of the event and the system that initiates the workflow.
+
+- **Message triggers.** You can start workflows by publishing to your broker or the Rhize broker.
+
+   If you want to initiate a workflow from a value changed in your data sources,
+   [create a rule]({{< relref "/how-to/publish-subscribe/turn-value-into-event" >}}).
+   As rules evaluate properties from equipment, they commonly are triggers from data emitted by level-1 and level-2 systems, such as a SCADA.
+
+   If you want to initiate a workflow by publishing to the Rhize broker, use a [message start event]({{< relref "bpmn-elements#message" >}}).
+   These messages often originate from level-3 and level-4 systems, such as an ERP.
+- **[Timer events]({{< relref "bpmn-elements#timer" >}}).** These work according to a schedule, either one-off or repeating.
+- **[API call]({{< relref "/how-to/gql/" >}})**. You can also start a BPMN workflow by sending a {{< abbr "mutation" >}} to the GraphQL API.
+  API triggers are commonly used in low-code workflows. For example, an operator may press a button to start a workflow and wait for its response.
+
+  To start BPMN workflows, Rhize has two primary API operations:
+
+     - `createAndRunBpmn` starts a workflow and does not wait for the response (asynchronous).
+     - `createAndRunBPMNSync` starts a workflow and waits for the process to complete or abort (synchronous).
+
+## Data processing and API calls
+
+Besides exchanging messages, BPMN workflows can also transform and evaluate payloads and make HTTP requests.
+All the data in a process is stored in a {{< abbr "process variable context" >}} that each element can access.
+
+The general logical elements are as follows:
+- To evaluate conditions, use a [gateway]({{< relref "/how-to/bpmn/bpmn-elements#gateways" >}})
```

Comment:
```suggestion
- To evaluate conditions, use an [exclusive gateway]({{< relref "/how-to/bpmn/bpmn-elements#gateways" >}})
```

Gateways allow more than just conditionals, so just adjusting this to be a bit more specific.